drivers/gpio/Kconfig

```diff
@@ -16,6 +16,7 @@ source "subsys/logging/Kconfig.template.log_config"
 
 config GPIO_SHELL
 	bool "GPIO Shell"
+	default y
```

Comment:
Why?

Reply:
This way you get GPIO shell automatically if you have GPIO and SHELL enabled.
Otherwise you have to enable it explicitly.

Most of the other shell implementations have done it this way.